### PR TITLE
Feature/fe/searchforms

### PR DESCRIPTION
**Pull request by elarens**
Minor fix (country options were generating hundreds of repetitive keys)

### CODE REVIEW

File: frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import {
   Checkbox, Collapse, Form, Typography, Space,
 } from 'antd';
 import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
```
Comment (bapower) [inline frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js]: I'm not sure why you are adding keys using uuidv4 considering it caused other bug?

**Review by bapower — COMMENTED** [review]
What is the PR for? The branch title isn't telling me what Jira story its related to?

File: frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js (hunk 1 above)
Comment (elarens) [reply frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js]: The pull request is to fix the bug caused by your change in the SearchForm. You were generating hundreds of similar keys in the Option loop (line ~105)

Comment (elarens) [reply frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js]: I will remove uuid from FacetedSearchGRoup and update the PR (I didn't really add them, they were just there)

Comment (elarens) [reply frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js]: Uuids are removed from the FacetedSearchGroup

**Review by bapower — APPROVED** [review]

**Pull request merged**